pkg/gitparse/gitparse_test.go

```diff
@@ -778,8 +780,13 @@ func assertDiffEqualToExpected(t *testing.T, expected *Diff, actual *Diff) {
 		assert.NoError(t, err)
 		assert.Equal(t, expectedDiffStr, actualDiffStr)
 	}
-	// TODO - Add test coverage for binary diffs (if it isn't already elsewhere)
 
+	if actual.IsBinary {
+		if actual.contentWriter != nil {
```

Comment:
It appears that for binary content, the actual content is always nil by default. 😕

Reply:
❓ Does the `git cat-file` command only return binary gibberish, or does it provide human-readable output?